packages/sdk-react/src/components/onboarding/hooks/useOnboardingEntity.ts

```
@@ -73,8 +73,30 @@ export function useOnboardingEntity(): OnboardingEntityReturnType {
+        ) {
+          const newSsnLast4 = individualPayload.id_number.slice(-4);
+          const existingEntityData = onboarding?.data?.entity;
+          const existingSsnLast4Value =
+            existingEntityData?.individual?.ssn_last_4?.value;
+
+          if (newSsnLast4 !== existingSsnLast4Value) {
+            individualPayload.ssn_last_4 = newSsnLast4;
+          }
+        }
+      } else if (fields.address?.country?.value !== 'US') {
+        delete (payload.individual as OptionalIndividualSchema).ssn_last_4;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)